lib/v2/dhu/news/xsxx.js

```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const baseUrl = 'https://news.dhu.edu.cn/6410';
+
+module.exports = async (ctx) => {
+    const { data: response } = await got(baseUrl);
+    const $ = cheerio.load(response);
+    const list = $('.news_UlA>li')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const a = item.find('a').first();
+            return {
+                title: a.text(),
+                link: `${baseUrl}${a.attr('href')}`,
+                pubDate: parseDate(item.find('p').text()),
+            };
+        });
```

Comment:
Please use the POST API of `https://news.dhu.edu.cn/_wp3services/generalQuery` instead.
docs: https://docs.rsshub.app/joinus/new-rss/before-start.html